public/content/roadmap/dencun/index.md

```diff
@@ -0,0 +1,151 @@
+- **Individual users** are always free to store their own copy of any data they would like for historical reference
+- **Rollup providers** are incentivized to store this data to improve the user experience of their rollup
+- **Block explorers** typically run archival nodes that will index and store all of this information for easy historical reference, made available to users through a web interface
+
+It is important to note, that recovering historical state is a **1-of-N trust modal**, meaning that you only need _a single honest actor_ to provide the data, and then _anyone can verify it_ to be correct using the current state of the network.
+
+## How does this upgrade contribute to the broader Ethereum roadmap? {#roadmap-impact}
+
+Proto-Danksharding sets the stage for the full implementation of [Danksharding](/roadmap/danksharding/), which splits up the burden of storing rollup data amongst node operators, so each only has to handle a small portion of the total. This will <!-- TIME-SENSITIVE --> enable significantly more data blobs per block, and help Ethereum grow by another order of magnitude.
+
+This is critical to [scale Ethereum to support billions of users](/roadmap/scaling/) with affordable fees and more advanced applications, while maintaining a decentralized network of participants. Without this, hardware requirements for node operators would continue to climb over time, requiring more-and-more expensive machinery to operate, pricing out users who would [otherwise participate](/run-a-node/). This would consolidate node operators to a select few who could afford the hardware, hindering decentralization.
+
+## Does this upgrade affect all Ethereum consensus and validator clients? {#client-impact}
+
+Yes. Proto-Danksharding (EIP-4844) involves changes to both the execution clients _and_ consensus clients. All production Ethereum clients have released updates to support the upgrade. Node operators must upgrade to a supported version to stay in sync with the chain after the upgrade. <!-- TIME-SENSITIVE --> [See details on supported client releases](https://blog.ethereum.org/2024/02/27/dencun-mainnet-announcement#client-releases)
+
+_Validator_ software is handled by the consensus clients, which have all been updated to accommodate the upgrade.
+
+## How does Cancun-Deneb (Dencun) affect Goerli or other Ethereum testnets? {#testnet-impact}
+
+- Devnets, Goerli, Sepolia and Holesky have all undergone the Dencun upgrade and have Proto-Danksharding fully functioning
+- Rollup developers can use these networks for EIP-4844 testing
+- Most users will be completely unaffected by this change to each testnet
+
+## What is the economic impact for the included Ethereum Improvement Proposals (EIPs)? {#economic-impact}
```

Comment:
It seems a bit narrow to focus on _only_ potential economic impacts, while there many other dimensions of impact to the upgrade.